manifests/m/Microsoft/AzureDataCLI/20.3.6/Microsoft.AzureDataCLI.locale.en-US.yaml

```diff
@@ -8,6 +7,7 @@ Publisher: Microsoft Corporation
 PackageName: Azure Data CLI
 License: Copyright (C) Microsoft Corporation
 ShortDescription: A command-line utility written in Python to bootstrap and manage the data services via REST APIs
+Moniker: azurecli

```

Comment:
Will this be too confusing with [Microsoft.AzureCLI](https://github.com/microsoft/winget-pkgs/blob/master/manifests/m/Microsoft/AzureCLI/2.30.0/Microsoft.AzureCLI.yaml#L5)?

Reply:
@Trenly thanks for calling that out. I was intending to update Microsoft.AzureCLI 🤦‍♂️